### ISSUE TITLE
OSADL情報のインポートについて

### ISSUE THREAD

**Issue by yyosioka** [KoukiHama/sw360#78]
[sw360-16.0.0-M1で動作確認を実施しているのですが
｢OSADL情報のインポート」に失敗します。

何か設定を変更する必要があるとか、バージョンアップすることで解決する等
情報がございましたらご教授いただけないでしょうか。

**Comment by KoukiHama** [comment]
はい、OSADLの方のデータ置き場が変わった影響で１６ではインポートできない状況です。定期的にOSADLのデータ置き場変更に合わせて変更加えてますが、最新のものがで正しくOSADL情報インポートできるか未確認です。
